sense-android-library/src/nl/sense_os/platform/SensePlatform.java

```diff
@@ -108,108 +88,10 @@ public SensePlatform(Context context) {
     public SensePlatform(Context context, ServiceConnection serviceConnection) {
         mServiceConnection = new SenseServiceConn(serviceConnection);
         mContext = context;
-        trivialSensorRegistrator = new TrivialSensorRegistrator(mContext);
         bindToSenseService();
     }
 
     /**
-     * Convenience method to add a data point without specifying the device UUID
```

Comment:
@orhan89 Indeed The aim-android-sdk will be merged with this branch and then we'll refactor it.